src/navigations/KeyboardWrapper.tsx

```diff
@@ -1,21 +1,15 @@
 import React from 'react';
 
 import {KeyboardAvoidingView, Platform} from 'react-native';
-import {useSafeAreaInsets} from 'react-native-safe-area-context';
 
 type Props = {
   children: React.ReactNode;
 };
 
 const KeyboardWrapper = ({children}: Props): JSX.Element => {
   const isIos = Platform.OS === 'ios';
-  const insets = useSafeAreaInsets();
-
   return (
-    <KeyboardAvoidingView
-      style={{flex: 1}}
-      keyboardVerticalOffset={insets.top}
-      behavior={isIos ? 'padding' : undefined}>
+    <KeyboardAvoidingView style={{flex: 1}} behavior={isIos ? 'padding' : undefined}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `useSafeAreaInsets` from the import statement and its usage in `KeyboardAvoidingView` might cause issues with the keyboard overlaying content on iOS devices. The `keyboardVerticalOffset` prop is used to adjust the height of the view when the keyboard is displayed, and it's typically set to the height of the device's status bar (which `useSafeAreaInsets().top` provides). Without this offset, the keyboard may cover some of the content.
```diff
-  const KeyboardWrapper = ({children}: Props): JSX.Element => {
-    const isIos = Platform.OS === 'ios';
+  import {useSafeAreaInsets} from 'react-native-safe-area-context';
+  
+  const KeyboardWrapper = ({children}: Props): JSX.Element => {
+    const isIos = Platform.OS === 'ios';
+    const insets = useSafeAreaInsets();

     return (
-     <KeyboardAvoidingView style={{flex: 1}} behavior={isIos ? 'padding' : undefined}>
+     <KeyboardAvoidingView
+       style={{flex: 1}}
+       keyboardVerticalOffset={insets.top}
+       behavior={isIos ? 'padding' : undefined}>
        {children}
      </KeyboardAvoidingView>
    );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->